trojsten/static/js/diploma_preview.js

```diff
@@ -0,0 +1,102 @@
+(function($)
+{
+
+    let fields = JSON.parse(template_fields.replace(/&quot;/g,'"'));
```

Comment:
Nie som si isty aku mame target skupinu browsra, ale bezne v trojsten webe pouzivame ES5 alebo transpilujeme do ES5 pomocou webpack alebo niecoho.

Ale kedze je to pre veducich tak to asi nevadi.